website/templates/emails/security_permissions_change.txt.mako

```diff
@@ -0,0 +1,25 @@
+Dear ${user.given_name},
+
+We are writing to let you know of a change to our permission model that may affect how you choose to organize your projects.
+
+Contributors on Open Science Framework (OSF) projects have one of three permission statuses: administrator, read+write, and read-only. These affect what actions those contributors can perform on the project. Projects can have sub-components that have contributor lists that are distinct from the project contributor list.
+
+Currently, project administrators can view the contents of components in their projects only if they are listed as contributors on those components. However, two weeks from today, project administrators will have read-only access to sub-components of their projects even if they are not listed as a contributor. This means that administrators can see the component and its contents.
+
+For example, if Thomas, Nikola, and Angus are administrators on project "AC/DC", but Angus is the sole contributor to a subcomponent "High Voltage" within that project, Thomas and Nikola will now be able to view the contents of "High Voltage".
+
+Overall, we expect this to provide a more intuitive experience when thinking about permissions within the OSF. Administrators can view the contents of their projects, even if they are not directly contributing to all components. Also, this is an important change because administrators can register projects (i.e., create permanent, uneditable versions). It is important that administrators know what they are registering.
+
+How will this affect you? If you are a contributor on a component of a project and do not want the administrator of that project to see the content, then you may wish to delete the component and move the content to a new project in which you are the administrator.
+
+Please do let us know if you have any questions or would like help in addressing the organization of your projects:
+
+support@osf.io
+
+Regards,
+
+The OSF Team
+Center for Open Science
+http://osf.io/
+
+Note: You are receiving this message because you have an account on the Open Science Framework. In system-wide security messages, we will not include links or ask you for information - passwords or otherwise.
```

Comment:
"In system-wide security messages, we will not ask you to click on links or ask you for information - passwords or otherwise."